neqo-transport/src/recovery/mod.rs

```diff
@@ -835,7 +840,9 @@ impl LossRecovery {
     /// When it has, mark a few packets as "lost" for the purposes of having frames
     /// regenerated in subsequent packets.  The packets aren't truly lost, so
     /// we have to clone the `SentPacket` instance.
-    fn maybe_fire_pto(&mut self, rtt: &RttEstimate, now: Instant, lost: &mut Vec<SentPacket>) {
+    fn maybe_fire_pto(&mut self, primary_path: &PathRef, now: Instant, lost: &mut Vec<SentPacket>) {
+        let path = primary_path.borrow();
+        let rtt = path.rtt();
```

Comment:
Why is this change needed? Compiles and tests succeed without it on my local machine.

Reply:
Might be a leftover from a previous change. Let me check.